HerPublicWebsite/Views/Questionnaire/IneligibleScotland.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+
+@model HerPublicWebsite.Models.Questionnaire.IneligibleScotlandViewModel
+@{
+    ViewBag.Title = "Sorry, this service isn't for you";
```

Comment:
Another silly thing that's just occurred to me - does gov uk have a style guide on apostrophes/quotes (i.e. should this be `isn’t` rather than `isn't`).

I remember the content people at ROH being very hot on this (and em-dashes vs en-dashes vs hyphens 😱)

Reply:
(If this is an issue, it'll need changing elsewhere as well)

Reply:
As discussed in Slack, leaving this here but will make a decision elsewhere